tests/test_image.py

```diff
@@ -0,0 +1,22 @@
+import unittest
+import os
+
+from pdfminer.image import BMPWriter
+
+
+class TestPngWriter(unittest.TestCase):
+
+    def test_write_bmp(self):
+        filename = os.path.join(
+            os.path.dirname(__file__),
+            'test_files/test_bmp.bmp'
+            )
+        fp = open(filename, 'wb')
+        bmp = BMPWriter(fp, 8, 10, 10)
+        bmp.write_line(0, b'0')
+        fp.close()
+        self.assertTrue(os.path.isfile(filename))
```

Comment:
Don't think we have access to a BMP parser, no time to write one

Reply:
Actually a BMP parser would file it as corrupted because technically speaking the content is not valid for a BMP, we're only testing if it can output a BMP at all

Reply:
alright, in that case i agree that's the best you can do. thanks for the clarification!